x-pack/plugin/security/build.gradle

```
@@ -91,7 +91,7 @@ dependencies {
   api "com.nimbusds:lang-tag:1.4.4"
   api "com.sun.mail:jakarta.mail:1.6.3"
   api "net.jcip:jcip-annotations:1.0"
-  api "net.minidev:json-smart:2.5.1"
+  api "net.minidev:json-smart:2.5.2"
   api "net.minidev:accessors-smart:2.4.2"
```

The `net.minidev:json-smart:2.5.2` [depends on](https://mvnrepository.com/artifact/net.minidev/json-smart/2.5.2) `net.minidev:accessors-smart:2.5.2` at compile time. Let's update `accessors-smart` version to keep them consistent.

```suggestion
  api "net.minidev:accessors-smart:2.5.2"
```